tv_apps/browser/js/awesomescreen.js

```diff
@@ -180,11 +180,10 @@ var Awesomescreen = {
       Browser.switchCursorMode(true);
     }).bind(this));
 
-    this.bookmark.addEventListener('loadDataByRange', (function(e){
+    this.bookmark.addEventListener('loadDataByNumber', (function(e){
```

Comment:
What does number mean here? does it mean how many records? how about loadMaxAmountData?